src/сomponents/Experimental/FirstType/FirstType.test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Adapter from 'enzyme-adapter-react-16';
+import { shallow, configure } from 'enzyme';
+import { shallowToJson } from 'enzyme-to-json';
+import FirstType from './FirstType';
+
+configure({adapter: new Adapter()});
+
+describe('FirstType', () => {
+  it('should render correctly', () => {
+    const output = shallow(
+      <FirstType>First title</FirstType>
+    );
+    expect(shallowToJson(output)).toMatchSnapshot();
+  });
+  it('should render correctly', () => {
```

Comment:
The assertion should have a unique name. To specify what are you testing

Reply:
Fixed, thank you!